diff --git a/backend/routes/generate_code.py b/backend/routes/generate_code.py
index d57d779..8361bf9 100644
--- a/backend/routes/generate_code.py
+++ b/backend/routes/generate_code.py
@@ -31,7 +31,7 @@ from prompts import create_prompt
 from prompts.claude_prompts import VIDEO_PROMPT
 from prompts.types import Stack
 
-# from utils import pprint_prompt
+from utils import pprint_prompt
 from ws.constants import APP_ERROR_WEB_SOCKET_CODE  # type: ignore
 
 
@@ -238,7 +238,7 @@ async def stream_code(websocket: WebSocket):
         )
         raise
 
-    # pprint_prompt(prompt_messages)  # type: ignore
+    pprint_prompt(prompt_messages)  # type: ignore
 
     ### Code generation
 
diff --git a/frontend/tailwind.config.js b/frontend/tailwind.config.js
index 2082387..ab6eb49 100644
--- a/frontend/tailwind.config.js
+++ b/frontend/tailwind.config.js
@@ -1,5 +1,7 @@
 /** @type {import('tailwindcss').Config} */
-module.exports = {
+import tailwindcssAnimate from "tailwindcss-animate";
+
+export default {
   darkMode: ["class"],
   content: [
     "./pages/**/*.{ts,tsx}",
@@ -74,5 +76,5 @@ module.exports = {
       },
     },
   },
-  plugins: [require("tailwindcss-animate")],
+  plugins: [tailwindcssAnimate],
 };
diff --git a/frontend/tsconfig.json b/frontend/tsconfig.json
index 4a83aa7..f5a8a20 100644
--- a/frontend/tsconfig.json
+++ b/frontend/tsconfig.json
@@ -5,6 +5,7 @@
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
+    "sourceMap": true,
 
     /* Bundler mode */
     "moduleResolution": "bundler",
