diff --git a/Dockerfile b/Dockerfile
index 06541d3..e970c2e 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -25,6 +25,8 @@ ARG ANTHROPIC_API_KEY
 ARG OPEN_ROUTER_API_KEY
 ARG GOOGLE_GENERATIVE_AI_API_KEY
 ARG OLLAMA_API_BASE_URL
+ARG OPENAI_LIKE_API_KEY
+ARG OPENAI_LIKE_API_BASE_URL
 ARG TOGETHER_API_KEY
 ARG TOGETHER_API_BASE_URL
 ARG VITE_LOG_LEVEL=debug
@@ -38,6 +40,8 @@ ENV WRANGLER_SEND_METRICS=false \
     OPEN_ROUTER_API_KEY=${OPEN_ROUTER_API_KEY} \
     GOOGLE_GENERATIVE_AI_API_KEY=${GOOGLE_GENERATIVE_AI_API_KEY} \
     OLLAMA_API_BASE_URL=${OLLAMA_API_BASE_URL} \
+    OPENAI_LIKE_API_KEY=${OPENAI_LIKE_API_KEY} \
+    OPENAI_LIKE_API_BASE_URL=${OPENAI_LIKE_API_BASE_URL} \
     TOGETHER_API_KEY=${TOGETHER_API_KEY} \
     TOGETHER_API_BASE_URL=${TOGETHER_API_BASE_URL} \
     VITE_LOG_LEVEL=${VITE_LOG_LEVEL} \
@@ -62,6 +66,8 @@ ARG ANTHROPIC_API_KEY
 ARG OPEN_ROUTER_API_KEY
 ARG GOOGLE_GENERATIVE_AI_API_KEY
 ARG OLLAMA_API_BASE_URL
+ARG OPENAI_LIKE_API_KEY
+ARG OPENAI_LIKE_API_BASE_URL
 ARG TOGETHER_API_KEY
 ARG TOGETHER_API_BASE_URL
 ARG VITE_LOG_LEVEL=debug
@@ -74,6 +80,8 @@ ENV GROQ_API_KEY=${GROQ_API_KEY} \
     OPEN_ROUTER_API_KEY=${OPEN_ROUTER_API_KEY} \
     GOOGLE_GENERATIVE_AI_API_KEY=${GOOGLE_GENERATIVE_AI_API_KEY} \
     OLLAMA_API_BASE_URL=${OLLAMA_API_BASE_URL} \
+    OPENAI_LIKE_API_KEY=${OPENAI_LIKE_API_KEY} \
+    OPENAI_LIKE_API_BASE_URL=${OPENAI_LIKE_API_BASE_URL} \
     TOGETHER_API_KEY=${TOGETHER_API_KEY} \
     TOGETHER_API_BASE_URL=${TOGETHER_API_BASE_URL} \
     VITE_LOG_LEVEL=${VITE_LOG_LEVEL} \
diff --git a/app/utils/constants.ts b/app/utils/constants.ts
index 240b4b9..370e005 100644
--- a/app/utils/constants.ts
+++ b/app/utils/constants.ts
@@ -401,7 +401,8 @@ async function getOpenAILikeModels(
   settings?: IProviderSetting,
 ): Promise<ModelInfo[]> {
   try {
-    const baseUrl = settings?.baseUrl || import.meta.env.OPENAI_LIKE_API_BASE_URL || '';
+    const baseUrl = settings?.baseUrl || import.meta.env.OPENAI_LIKE_API_BASE_URL;
+    logger.info('OpenAILike baseUrl:', baseUrl);
 
     if (!baseUrl) {
       return [];
@@ -413,6 +414,9 @@ async function getOpenAILikeModels(
       apiKey = apiKeys.OpenAILike;
     }
 
+    apiKey = apiKey || import.meta.env.OPENAI_LIKE_API_KEY;
+    logger.info('OpenAILike apiKey:', apiKey);
+
     const response = await fetch(`${baseUrl}/models`, {
       headers: {
         Authorization: `Bearer ${apiKey}`,
